acme/acme/client_test.py

```diff
@@ -21,7 +21,6 @@
 KEY = jose.JWKRSA.load(test_util.load_vector('rsa512_key.pem'))
 KEY2 = jose.JWKRSA.load(test_util.load_vector('rsa256_key.pem'))
 
-
```

Comment:
Looks like you introduced a spurious diff here; could you restore the deleted line?